### ISSUE TITLE
Collection activity: Add sorting

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#5522]
### Is your feature request related to a problem?

I was checking [this](https://beta.kodadot.xyz/rmrk2/collection/3e6f301b0a0b1a0e4e-MRWH/activity) collection. Open collection - Activity. Can we please add sorting options here? So we can sort by event, addresses, amount..... To make it more interactive. 
<img width="1430" alt="Screenshot 2023-04-05 at 09 52 50" src="https://user-images.githubusercontent.com/104380776/230016993-89b74dde-c084-43d0-8fd5-12f0e8b489b0.png">

Also, add some kind of sorting here: 

<img width="477" alt="Screenshot 2023-04-05 at 09 55 15" src="https://user-images.githubusercontent.com/104380776/230017468-767d932c-7c34-46b2-b9ca-d0c4db4bb547.png">

### Describe the solution you would like

New sorting

### Describe alternatives you have considered

-

**Comment by yangwao** [comment]
we've got showing by events for now on left

https://user-images.githubusercontent.com/5887929/230057513-df123a21-5ef9-4924-b6d8-785f98d00b2c.mov

and activity stream is sorted by newest occurrence on top 

> add some kind of sorting here:

what kind of sorting you would like to see here?

**Comment by JustLuuuu** [comment]
> we've got showing by events for now on left

Yeah, this is good. But you cannot sort it for example by name (to see what transactions were happening on one NFT),  highest amount (if you want to see the highest sales), or by addresses (the artist wants to check if someone owns more than 1 NFT and in the same time see pictures of the NFTs). My idea was just to add sorting arrows here to make it more interactive:
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/104380776/230064725-63c9ba96-9950-452f-9487-ca58bd403243.png">

I know most of these things can be found somewhere else if they search it. But here it would be nice to have everything in one place :)

> What kind of sorting would you like to see here?

I realized it's already sorted in a good way (not flippers because they are not working)

**Comment by yangwao** [comment]
> highest amount (if you want to see the highest sales), or by addresses (the artist wants to check if someone owns more than 1 NFT and in the same time see pictures of the NFTs).

yup we want to keep it basic here 🤐

- https://github.com/kodadot/ops-internal/issues/290

> (not flippers because they are not working)

yup, seems some more crafting ⚒️ 
cc @daiagi 👀

**Comment by daiagi** [comment]
Flippers are working

See here for example
https://beta.kodadot.xyz/rmrk/collection/0a24c7876a892acb79-KUBIBITS/activity?page=1


But it is missing some default display for case where there are no flippers
@exezbcz

**Comment by exezbcz** [comment]
@JustLuuuu 
> to see what transactions were happening on one NFT)

for one nft is gallery detail

> the artist wants to check if someone owns more than 1 NFT and in the same time see pictures of the NFTs

- you can click on holder and see more options - also with pictures of the nfts

- not sure if we should implement this type of sorting 

> My idea was just to add sorting arrows here to make it more interactive:

- Yes, it would be nice. I had it planned out for the gallery item's activity, but it wasn't implemented, so maybe we can try again here

**Comment by exezbcz** [comment]
@daiagi Yes, when there are no flippers, I will create some copy to display.

**Comment by exezbcz** [comment]
Looks like it's quiet in here!

something like this? @daiagi 
- formatting same as in explore empty state
![image](https://user-images.githubusercontent.com/90852205/230133370-a52e85c5-4d1d-43a9-8076-cdfda67acd83.png)



guess you need one for holders as well no? Or there is always the mint null address right

**Comment by daiagi** [comment]
I don't think we need one for holders. Nfts are always held by someone

By the way @exezbcz , I propose we change the definition of a flipper.
Right now, if the minter had sold, he is considered flipper
I think it should only be a person that bought and then sold

Wdyt?

**Comment by exezbcz** [comment]
@daiagi yeah, agree, if there is any way how to distinct minter from other flippers than go on
